katsdptelstate/rdb_reader.py

```diff
@@ -49,23 +50,33 @@ def end_sorted_set(self, key):
         self._zset = {}
 
 
-def load_from_file(client, filename):
+def load_from_file(callback, file):
     """Load keys from the specified RDB-compatible dump file.
 
     Parameters
-    ---------
-    client : :class:`redis.StrictRedis`-like
-        Redis client wrapper
-    filename : str
-        Filename of .rdb file to import
+    ----------
+    callback : :class:`rdbtools.RdbCallback`-like
+        Backend-specific callback that stores keys as RDB file is parsed
```

Comment:
It's a bit stricter than that: it has to have an `n_keys` member.

Reply:
Good point. Of course there is now also a faint whiff, since the caller could examine `callback.n_keys` instead and then `load_from_file` does not need to know about it. But then that increases the line count by at least 2 per call to `load_from_file` so not quite worth it. :-)